content/syllabuses/short-course-datascience-1.md

```diff
@@ -5,35 +5,122 @@ title: Datascience short course 1
 
 This course is to be completed in 8 weeks at maximum 10 hours per week. 
 
-Learners doing this course should first do the "Intro to Tilde for tech bootcamps" course.
+Learners doing this course should first do the "Intro to Tilde for non-coder bootcampers" course.
+
+
+## Week 1: Introduction to Data Science. The big picture
+
+### Level 1: Conversational fluency 
+
+- {{< contentlink path="medium-touch-course-common/0-how-this-course-works" >}}
+- {{< contentlink path="data-science/foundational-short-course/1-what-is-data-science" >}}
+- {{< contentlink path="data-science/foundational-short-course/2-the-data-science-method" >}}
+- {{< contentlink path="topics/git/git-summary">}}
```

Comment:
Should these git things not go after "why python and git"? If we just tell people to do this weird thing that doesn't look like datascience without telling them why it matters then they might get a bit confused. Some people are even resistant to learning git. Letting them know why up front reduces friction